packages/pwa-kit-mcp/src/tools/developer-guideline.js

```diff
@@ -106,9 +106,9 @@ This document offers guidelines in the development of Salesforce Commerce Compos
 `
 
 export default {
-    name: 'development_guidelines',
-    description: `You must follow this development guidelines before attempting to analyze/ generate / refactor / modify / fix code.
-    - e.g. "Create a customer service Chat component", "Find bugs in my_script.jsx", "Refactor my_script.jsx to use React Hooks"`,
+    name: 'get_development_guidelines',
+    description: `You must follow these development guidelines before attempting to analyze, generate, refactor, modify, or fix code.
```

Comment:
This is my fault as I missed this in my edit. I think it would be more helpful to say:
```suggestion
    description: `You must follow the PWA Kit development guidelines before attempting to analyze, generate, refactor, modify, or fix code.
```